nix/nim-status/build.nix

```diff
@@ -0,0 +1,126 @@
+{ pkgs, stdenv, lib, fetchFromGitHub
+# Dependencies
+, xcodeWrapper
+, git, androidPkgs
+, platform ? "android"
+, arch ? "386"
+, api ? "23" }:
+
+let
+  inherit (lib) attrNames getAttr strings concatStringsSep concatMapStrings;
```

Comment:
The only thing used from here is `getAttr`, which might just as well be called as `lib.getAttr` and this line dropped.